applications/templates/kube-system/02-appproject.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: kube-system
+spec:
+  clusterResourceWhitelist:
+  - group: rbac.authorization.k8s.io
+    kind: ClusterRoleBinding
+  - group: rbac.authorization.k8s.io
+    kind: ClusterRole
+  - group: apiextensions.k8s.io
+    kind: CustomResourceDefinition
+  - group: apiregistration.k8s.io
+    kind: APIService
+  sourceRepos:
+    - https://github.com/helm/charts.git
+  destinations:
+    - server: https://kubernetes.default.svc
+      namespace: kube-system
+  description: "Applications that support kubernete operations"
```

Comment:
`kubernete` typo, missing end line and I would simply remove the description

Also, I started to think that it would be a good practice to add a `README.md` to each "sub" chart to explain people what it does, why it's used, additional info... Example [bootstrap](https://github.com/edgelevel/gitops-k8s/tree/master/bootstrap)

Reply:
👍